### DISCUSSION TITLE
Set USB transfer mode to MTP on GoPro Hero 9 Black with broken rear screen

### DISCUSSION THREAD

**Discussion by gustavokklein** [gopro/labs#1191] [Q&A]
Hey guys! I have a GoPro Hero Black 9 with a broken rear screen. I usually use SD card adapters to transfer files, but I've had the card corrupt a few times during the process because these adapters are crap.

Is there any way to change the USB mode using a QR code?

Or any other way?

**Comment by dnewman-gpsw** [comment]
There is no QR code command for that.

**Comment by gustavokklein** [comment]
But is there any other way to do this without needing a display?

**Comment by dnewman-gpsw** [comment]
No, sorry, not that I know of.
